components/dashboard/history-dashboard.tsx

```
@@ -102,32 +110,39 @@ function getPresentationSlides(raw: any): any[] {
   const content = doc.content || {};
   const metadata = doc.metadata || content.metadata || {};
-  
+
   switch (doc.type) {
-    case 'resume':
-      return content.resumeData?.name || content.name || 'Resume';
-    case 'presentation':
+    case "resume":
+      return content.resumeData?.name || content.name || "Resume";
+    case "presentation":
       const slides = getPresentationSlides(doc);
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)